census/estimator/trainer/task.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
If we can add some Markup text/Documentation in terms what the Notebook is doing will be great

Reply:
If the original .py file has top level comments they will be turned into markdown cells, so the burden is on the .py file to have better documentation and comments.  See for example the census/keras notebook i just added.